### DISCUSSION TITLE
Thoughts on deprecating jsonlines output?

### DISCUSSION THREAD

**Discussion by prabhu** [owasp-dep-scan/dep-scan#211] [Ideas]
Depscan generates a report in jsonlines format. While this format was used to import results directly into Google BigQuery, it is not clear if anyone still uses this. Can we deprecate and remove this format and go all-in with CycloneDX VDR?

## Related tickets

https://github.com/owasp-dep-scan/dep-scan/issues/158

**Comment by marcelstoer** [comment]
Would keeping it cause any specific troubles? If not, then I'd rather keep it.

**Reply by prabhu** [reply]
Keeping it is increasing maintenance effort. Most users have switched to the VDR format, which is a far better choice, to be honest.
